lib/routes/sohu/mobile.ts

```diff
@@ -0,0 +1,109 @@
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['m.sohu.com/limit'],
+            target: '/mobile',
+        },
+    ],
+    name: '首页新闻',
+    maintainers: ['asqwe1'],
+    handler,
+    description: '订阅手机搜狐网的首页新闻',
+};
+
+async function handler() {
+    try {
+        const response = await ofetch('https://m.sohu.com/limit', {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 15_0 like Mac OS X)'
+            }
+        });
+        // 从HTML中提取JSON数据
+        const $ = cheerio.load(response);
+        const jsonScript = $('script:contains("WapHomeRenderData")').html();
+        const jsonMatch = jsonScript?.match(/window\.WapHomeRenderData\s*=\s*({.*})/s);
+        if (!jsonMatch?.[1]) {
+            throw new Error('WapHomeRenderData 数据未找到');
+        }
+        const renderData = JSON.parse(jsonMatch[1]);
+        const list = extractPlateBlockNewsLists(renderData)
+            .map((item) => ({
+                title: item.title,
+                link: new URL(item.link || item.url, 'https://m.sohu.com').href,
+            }))
+            .filter((item) => item?.title && item?.link);
+        const items = await Promise.all(
+            list.map((item) =>
+                cache.tryGet(item.link, async () => {
+                    try {
+                        const detailResp = await ofetch(item.link);
+                        const $d = cheerio.load(detailResp);
+                        return {
+                            ...item,
+                            description: $d('#articleContent').first().html(),
+                            pubDate: parseDate($d('.time').text()) // 添加时间解析
+                        };
+                    } catch (error) {
+                        logger.error(`获取详情失败: ${item.link}`, error);
+                        return item;
+                    }
+                })
+            )
+        );
+        return {
+            title: '手机搜狐新闻',
+            link: 'https://m.sohu.com/limit',
+            item: items,
+        };
+    } catch (error) {
+        logger.error('抓取失败:', error);
+        return {
+            title: '手机搜狐新闻',
+            link: 'https://m.sohu.com/limit',
+            item: [],
+        };
+    }
+}
+
+function extractPlateBlockNewsLists(jsonData: any) {
+    const result: any[] = [];
+    for (const key of Object.keys(jsonData)) {
+        if (key.startsWith('PlateBlock')) {
+            const plateBlock = jsonData[key];
+            // 处理新闻列表
+            if (plateBlock?.param?.newsData?.list) {
+                result.push(...plateBlock.param.newsData.list);
+            }
+            // 处理焦点图数据
+            if (plateBlock?.param?.focusData?.list) {
+                result.push(...plateBlock.param.focusData.list);
+            }
```

Comment:
```suggestion
            }
            if (plateBlock?.param?.feedData0?.list) {
                result.push(...plateBlock.param.feedData0.list);
            }
            if (plateBlock?.param?.feedData1?.list) {
                result.push(...plateBlock.param.feedData1.list);
            }
```
Those exist in `PlateBlockHealth` and `PlateBlockTravel`